src/components/DataTypes/date-time.tsx

```diff
@@ -1,10 +1,14 @@
 import { JSONSchema6 } from 'json-schema';
 import assign = require('lodash/assign');
+import memoize = require('lodash/memoize');
+import * as moment from 'moment';
 import * as React from 'react';
 import { DataTypeEditProps, InputProps } from 'rendition';
 import * as utils from '../../utils';
 import Input from '../Input';
 
+(moment as any).suppressDeprecationWarnings = true;
```

Comment:
What deprecation messages are these? Why do we need to depend on deprecated functionality?

Reply:
I was trying to clean up test output a little bit, on reflection its best to leave this in!